classes/class-nets-easy-assets.php

```diff
@@ -109,11 +114,68 @@ public function inject_iframe_script() {
 		<?php
 	}
 
+	/**
+	 * Loads script for the embedded inline flow.
+	 *
+	 * @return void
+	 */
+	public function nexi_inline_js() {
```

Comment:
I know you are following how function names are called previously here, but i think we should rename this to say what it does. Like `enqueue_nexi_inline_js`